test/name_collisions_test.rb

```diff
@@ -0,0 +1,160 @@
+require 'test_helper'
+$VERBOSE=nil #otherwise we get lots of C:/gems/surrounded/lib/surrounded/context/name_collision_detector.rb:25: warning: instance variable @handler not initialized
```

Comment:
This is intentionally turned on in the Rakefile https://github.com/saturnflyer/surrounded/blob/master/Rakefile#L8